modules/tls-profiles-understanding.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * security/tls-security-profiles.adoc
+
+[id="tls-profiles-ingress-understanding_{context}"]
+= Understanding TLS security profiles
+
+You can use a TLS (Transport Layer Security) security profile to define which TLS ciphers are required by various {product-title} components.
+
+// A TLS security profile defines the TLS ciphers that the Kubernetes API server must use when connecting with the kubelet to protect communication between the kubelet and the Kubernetes API server.
+
+You can specify one of the following TLS security profiles:
+
+.TLS security profiles
+[cols="1,2a",options="header"]
+|===
+|Profile
+|Description
+
+|`Old`
+|This profile is intended for use with legacy clients or libraries. The profile is based on link:https://wiki.mozilla.org/Security/Server_Side_TLS#Old_backward_compatibility[recommended configurations].
+
+The `Old` profile requires a minimum TLS version of 1.1.
+
+// TODO: according to the oc explain docs, this is 1.0 ^
+
+|`Intermediate`
+|This profile is the recommended configuration for the majority of clients. It is the  default TLS security profile for the kubelet and Kubernetes control plane. The profile is based on link:https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28recommended.29[recommended configurations].
```

Comment:
kubelet, the control plane and the routers, maybe?

Reply:
Yep, I just confirmed that it is the default profile for all components, thanks!